project-base/app/src/FrontendApi/Model/Cart/MergeCartFacade.php

```diff
@@ -42,6 +42,23 @@ public function mergeCartByUuidToCustomerCart(string $cartUuid, CustomerUser $cu
         $this->mergeCarts($oldCart, $customerCart);
     }
 
+    /**
+     * @param string $cartUuid
+     * @param \App\Model\Customer\User\CustomerUser $customerUser
+     */
+    public function overwriteCustomerCartWithCartByUuid(string $cartUuid, CustomerUser $customerUser): void
+    {
+        $oldCart = $this->cartFacade->getCartCreateIfNotExists(null, $cartUuid);
+        $customerCart = $this->cartFacade->getCartCreateIfNotExists($customerUser, null);
+
+        $oldCart->assignCartToCustomerUser($customerUser);
+        $oldCart->setModifiedNow();
```

Comment:
I would move this call into `assignCartToCustomerUser` to avoid temporal coupling (I knew it was a thing but I  had to look up the term, TBH :smile: )

Reply:
Assuming you mean the line with:
`$oldCart->setModifiedNow();`
I did that